api/test/integration/test_manager_endpoints.tavern.yaml

```diff
@@ -1241,6 +1241,21 @@ stages:
           total_affected_items: 0
           total_failed_items: 1
 
+  # PUT /manager/configuration
+  - name: Try to upload an invalid configuration with an invalid content-type
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/manager/configuration"
+      method: PUT
+      data: "{invalid_ossec_conf:s}"
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+    response:
+      status_code: 406
+      json:
+        error: 6002
+
```

Comment:
It would be interesting to add a test like this but with a valid configuration. In this way, we set the behavior and in both cases, we establish that regardless of the content of the body, if its content-type is wrong, this will be indicated through the API response.

Reply:
The content-type is checked before getting into the framework, in the middlewares. It does not matter what type of content we receive.

Reply:
The content-type is checked before getting into the framework, in the middlewares. It does not matter if the body is valid or not.